addon/services/notifications.js

```diff
@@ -23,11 +23,12 @@ export default Service.extend({
       throw new Error("No notification message set");
     }
 
+    const defaultAutoClear = typeof globals.autoClear === 'boolean' ? globals.autoClear : false;
     const notification = EmberObject.create({
       message: options.message,
       type: options.type || 'info',
-      autoClear: (isEmpty(options.autoClear) ? getWithDefault(globals, 'autoClear', false) : options.autoClear),
-      clearDuration: options.clearDuration || getWithDefault(globals, 'clearDuration', 3200),
+      autoClear: (isEmpty(options.autoClear) ? defaultAutoClear : options.autoClear),
+      clearDuration: options.clearDuration || globals.clearDuration || 3200,
```

Comment:
Would `globals.clearDuration` ever be `0`? I suppose in that case this PR changes the behavior (overwrites it with 3200).

Reply:
ah that's a good point 🤔  I guess maybe someone could define it as `0` for tests so I guess this is technically different then

Do you want to try a PR using nullish coalescing here and I can release it as another patch? 👍

Reply:
Sure, I'll throw one up here later today if you haven't already fixed it by then.